frontend/src/agenda/agenda.vue

```diff
@@ -1,13 +1,12 @@
 <template>
   <div>
-    <div class="inline-title">{{ title }}</div>
     <keep-alive>
-      <full-calendar ref="qcalendar"
-                     key="qcalendar"
-                     class="calendar-margins"
+      <full-calendar ref="acalendar"
```

Comment:
Why did this change to `acalendar`? `qcalendar` sounded like a reasonable naming decision before

Reply:
sorry... they're not the same calendar.  technically the refs should be destryed immediately upon changing components but I didn't want to mess around.  easy enough just to give the other component its own ref.  i went with acalendar just because its the agenda calendar